src/ziggurat/ssl/properties.clj

```diff
@@ -0,0 +1,44 @@
+(ns ziggurat.ssl.properties
+  (:require [ziggurat.config :refer [ziggurat-config]]
+            [ziggurat.ssl.constants :as const])
+  (:import [org.apache.kafka.streams StreamsConfig]
+           [org.apache.kafka.common.config SslConfigs SaslConfigs]))
+
+(defn- ssl-configs
+  [key]
+  (get-in (ziggurat-config) [:ssl-config key]))
+
+(defn- validate-mechanism
+  [mechanism]
+  (when-not (contains? #{const/mechanism-plain const/mechanism-scram-512} mechanism)
+    (let [message (format "SSL mechanism can either be %s or %s" const/mechanism-plain const/mechanism-scram-512)]
+      (throw (IllegalArgumentException. message {:mechanism mechanism})))))
+
+(defn- validate-protocol
+  [protocol]
```

Comment:
Since ziggurat is open source, we should remove validations of any kind. Since kafka will eventually validate the properties on its side, we can remove this code.